src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/util/RecommendationToPartADataMapper.kt

```diff
@@ -16,7 +17,25 @@ class RecommendationToPartADataMapper {
       val recallType = findRecallTypeToDisplay(recommendation.data.recallType)
       val isThisAnEmergencyRecall = convertBooleanToYesNo(recommendation.data.isThisAnEmergencyRecall)
 
-      return PartAData(custodyStatus, recallType, recommendation.data.responseToProbation, isThisAnEmergencyRecall)
+      val selectedAlternativesMap = convertToMap(recommendation.data.alternativesToRecallTried?.selected)
+
+      return PartAData(custodyStatus, recallType, recommendation.data.responseToProbation, isThisAnEmergencyRecall, selectedAlternativesMap)
+    }
+
+    private fun convertToMap(selectedAlternatives: List<SelectedAlternative>?): Map<String, String> {
+      val selectedAlternativesMap = selectedAlternatives?.associate { it.value to it.details } ?: emptyMap()
+      return mapOf(
+        "warning_letter_details" to (selectedAlternativesMap["WARNINGS_LETTER"] ?: ""),
```

Comment:
You can use https://github.com/ministryofjustice/make-recall-decision-api/blob/main/src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/util/MrdTextConstants.kt#L6 here maybe

Reply:
Done